.github/workflows/tests.yaml

```diff
@@ -15,3 +15,7 @@ jobs:
           python -m pip install --upgrade pip
           pip install -r requirements.txt
       - run: pytest -vv
+      - name: Install Telegraf
+        run: sudo apt-get install -y telegraf
+      - name: Test InfluxDB output
+        run: telegraf --test --config slo.conf
```

Comment:
Good idea to add a CI check. Will this command actually return non-zero in case an error occurred?

Reply:
Yes. It was failing before I introduced the fix